exporter/awsemfexporter/metric_translator.go

```diff
@@ -82,6 +82,7 @@ type CWMetricStats struct {
 // 	- pdata.DoubleDataPointSlice
 // 	- pdata.IntHistogramDataPointSlice
 // 	- pdata.DoubleHistogramDataPointSlice
+//  - pdata.DoubleSummaryDataPointSlice
```

Comment:
Align with previous comment?

Reply:
Sure. Didn't see this in the IDE. I will replace all tabs with spaces :)